README.md

```diff
@@ -84,8 +84,13 @@ Find a list of all the available options for your DogStatsD Client in the [Datad
 
 ### Supported environment variables
 
-* If the `addr` parameter is empty, the client uses the `DD_AGENT_HOST` environment variables to build a target address.
-  Example: `DD_AGENT_HOST=127.0.0.1:8125` for UDP, `DD_AGENT_HOST=unix:///path/to/socket` for UDS and `DD_AGENT_HOST=\\.\pipe\my_windows_pipe` for Windows named pipe.
+* If the `addr` parameter is empty, the client will:
+  * First use the `DD_DOGSTATSD_URL` environment variables to build a target address. The url can start with `udp://` to connect using UDP or with `unix://` to use a Unix Domain Socket.
+    Example for UDP url: `DD_DOGSTATSD_URL=udp://localhost:8125`
+    Example for UDS: `DD_DOGSTATSD_URL=unix:///var/run/datadog/dsd.socket`
+    Example for Windows named pipe`DD_AGENT_HOST=\\.\pipe\my_windows_pipe`
+  * Fallback to the `DD_AGENT_HOST` environment variables to build a target address.
+    Example: `DD_AGENT_HOST=127.0.0.1:8125` for UDP, `DD_AGENT_HOST=unix:///path/to/socket` for UDS and `DD_AGENT_HOST=\\.\pipe\my_windows_pipe` for Windows named pipe.
```

Comment:
Is the omission of `DD_AGENT_PORT` intentional here? Is it semi-deprecated?

Reply:
It was never documented in the `READMe.md`. I'll add it with the refactor to remove the `goto`.